src/traces/parcoords/parcoords.js

```diff
@@ -557,7 +557,7 @@ module.exports = function(root, svg, styledData, layout, callbacks) {
                     .outerTickSize(2)
                     .ticks(wantedTickCount, d.tickFormat) // works for continuous scales only...
                     .tickValues(d.ordinal ? // and this works for ordinal scales
-                        sdom.map(function(d, i) {return texts && texts[i] || d;}) :
+                        sdom.map(function(d, i) {return texts && texts[i] !== undefined ? texts[i] : d;}) :
```

Comment:
I think you guys get this but just to be clear, there's a definite use case for `tickvals` without `ticktext` at all - when you want ticks at specific positions but labeled as regular numbers - see eg https://github.com/plotly/plotly.js/blob/master/test/image/mocks/axes_enumerated_ticks.json - this one has both completely missing `ticktext` (for the y axis) and shorter `ticktext` (for the x axis, ticks at 4 and 5 have no corresponding text). Some things to note about [the current cartesian implementation](https://github.com/plotly/plotly.js/blob/f42802689a99cba4057434664853e848c75625d6/src/plots/cartesian/axes.js#L878-L879) though:
- It stringifies any `ticktext[i]` it finds that's not `undefined` - including `null`, so it matches what you have in this PR. Does this count as a bug? Perhaps we should filter out `null`, as we've generally taken that to be indistinguishable from `undefined` within the json (but not within `restyle/relayout` calls so it can be tricky to even insert a `null` value...) but I would vote to leave numbers and even booleans to be stringified - it's kind of weird, but I feel like the user's intent is clear in those cases.
- It doesn't *just* stringify the number if no `ticktext[i]` is found - it formats it similarly to regular numeric ticks, just with potentially [more digits](https://github.com/plotly/plotly.js/blob/f42802689a99cba4057434664853e848c75625d6/src/plots/cartesian/axes.js#L780) than automatic ticks get, since these are not guaranteed to be round numbers. But it follows all the other formatting you might have provided: `exponentformat`, `tickprefix`, etc etc. I haven't looked at parcoords in enough detail to know if there is equivalent subtlety here, but if there is it should be included.